tests/accessor/generic_accessor_api_common.h

```diff
@@ -58,8 +58,23 @@ void test_accessor_methods(const AccT &accessor,
 #endif
 }
 
+template<typename Accessor>
+extern const sycl::target accessor_target_v;
```

Comment:
Why this `extern` definition?

Reply:
It's the base template that's being specialized below. For a variable template marking it extern allows me to omit the (invalid) definition and cause a compiler error when it's used in a constexpr context.

Reply:
Why not. :-)